pavelib/assets.py

```diff
@@ -718,6 +705,28 @@ def execute_compile_sass(args):
         )
 
 
+def get_parsed_option(options, name, default=None):
+    """
+    Options to a command are provided as string value. Convert it to list where a list is expected
+    after fetching an option specified by name
+    Args:
+        options: Command line arguments passed via paver command.
+        name: name of option to parse
+        default: if option not in options, return default
+    """
+    option = default
+    try:
+        option = getattr(options, name)
+        if isinstance(option, basestring):
+            option = option.split(',')
+        else:
+            option = option if isinstance(option, list) else [option]
+    except AttributeError:
+        pass
```

Comment:
That's done to avoid unnecessary if,else blocks. Purpose here is not to raise exception but to check if we can get value from `getattr(options, name)` . If `name` doesn't exist in `options`,  exception will be raised. `option`will remain same i.e set to `default` in start of method and in `finally` will eventually be returned.